src/main/java/com/denizenscript/depenizen/bukkit/events/shopkeepers/ShopKeeperTradeCompletedScriptEvent.java

```diff
@@ -0,0 +1,57 @@
+package com.denizenscript.depenizen.bukkit.events.shopkeepers;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.depenizen.bukkit.objects.shopkeepers.ShopKeeperTag;
+import com.nisovin.shopkeepers.api.events.ShopkeeperTradeCompletedEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class ShopKeeperTradeCompletedScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // shopkeeper trade
+    //
+    // @Warning This event should not be cancelled. Consider using <@link event ShopKeeperTradeInitiatedScriptEvent> instead.
```

Comment:
Don't think this needs a full on warning, maybe just not having the `@Cancellable` meta anymore & a link to the other event with a note to use that if you want to cancel?
Perhaps also a warning if cancelled in `#cancellationChanged` if you want to, but not required.